builder/controllers/makeImageController.go

```diff
@@ -0,0 +1,56 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+	"github.com/gin-gonic/gin"
+)
+
+func MakeDockerImage(ctx *gin.Context) {
+	cli, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
+	if err != nil {
+		 ctx.String(500, "Error creating Docker Client")
+		 return
+	}
+
+	// Build an image from a Dockerfile in the current directory
+	buildcontext, err := os.Open("./model.tar")
```

Comment:
wasn't the files zip?

Reply:
We need to unzip and add the Dockerfile inside. I just made a tar after adding a Dockerfile.

Reply:
where are we adding the docker image and getting the tar file?